### ISSUE TITLE
query ("webView marked:'XXX'") doesn't work

### ISSUE THREAD

**Issue by christianbader** [calabash/calabash-ios#85]
it seems that "marked:" is not working properly for UIWebViews.

The following is working fine:

irb(main):004:0> query "webView", :accessibilityLabel
=> ["Web View"]

but:

query ("webView marked:'Web View'")
=> []

just logs an empty array.

**Comment by TitouanVanBelle** [comment]
Has this issue been fixed? I'm using 0.16.4 and I still face the same issue.

```
(byebug) query "webView", :accessibilityLabel
["OfferWall_WebView"]
(byebug) query ("webView marked:'OfferWall_WebView'")
[]
(byebug) query ("webView accessibilityLabel:'OfferWall_WebView'")
[]
(byebug) query("webView {accessibilityLabel = 'OfferWall_WebView'}")
[{"alpha"=>1, "enabled"=>true, "id"=>nil, "visible"=>1, "frame"=>{"y"=>0, "x"=>0, "width"=>320, "height"=>568}, "accessibilityElement"=>false, "class"=>"UIWebView", "label"=>"OfferWall_WebView", "description"=>"<UIWebView: 0x7fd2961cd050; frame = (0 0; 320 568); layer = <CALayer: 0x7fd2961ac740>>", "value"=>nil, "rect"=>{"y"=>0, "center_x"=>187.5, "center_y"=>332.81, "x"=>0, "width"=>320, "height"=>568}}]
(byebug) 

```

**Comment by jmoody** [comment]
Duplicate of: https://github.com/calabash/calabash-ios/issues/735 which is still open.

It is hard to remove this because it will break backward compatibility.  I feel your pain.
